README.md

```diff
@@ -1,5 +1,9 @@
 ﻿# Z-Wave Home Automation library for .NET
 
+## About this fork
+I've forked this to remove NLog as the project I am using this library in doesn't use NLog thus adding more libraries seem useless.
+This fork also contains WIP's for adding new commandclasses to the library when I stumble upon one that's missing.
+
```

Comment:
If the intention is to merge into the main repo, this note should be here. Also removal of NLog is not a good thing. There are other projects already using this library that also relay on NLog.
So adding *CentralScene CommandClass* is ok as long the PR doesn't involve structural changes to the ZWaveLib project that have not been discussed.